discovery-provider/src/tasks/index_aggregate_user.py

```diff
@@ -400,13 +396,22 @@ def update_aggregate_table(
                 )
 
             if is_refreshed:
-                redis.set(
-                    index_aggregate_user_last_refresh_completion_redis_key,
-                    int(time.time()),
+                session.execute(
```

Comment:
I think we can re-evaluate how we refresh this. Originally, we wanted to larger refreshes every two weeks to ensure data accuracy since we were depending on redis and postgres. Now, that we can do everything in a single transaction it's probably best to keep it simple and just refresh with maybe a slight overlap each time. Then, we don't need to write elapsed time in indexing checkpoints, just keep track of the blocknumber.